src/sam.json

```diff
@@ -0,0 +1,70 @@
+{
+    "id": "required",
+    "schemaVersion": "required",
+    "name": "required",
+    "projectType": "required",
+    "boundedContexts": {
+        "REQUIRED": "required",
+        "LISTOF": {}
+    },
+    "teams": {
+        "REQUIRED": "required",
+        "LISTOF": {}
+    },
+    "sumoLogicSourceCategories": {
+        "REQUIRED": "required",
+        "LISTOF": {}
+    },
```

Comment:
What happens when these are required but you don't have one...like 1UP doesn't use sumologic at all.

Reply:
The way this one is, you'd need a section, but it could be empty.

Reply:
Is that documented in the readme?

Reply:
I guess it might be nice to have another example in the readme that shows how that would happen...or a section in the readme specifically saying how the sam.yaml behaves and what it expects.

Reply:
I have an example of this spec in the README. I'll try to make it clear how this works.